### ISSUE TITLE
How to maintain raw dataset

### ISSUE THREAD

**Issue by LienReyserhove** [trias-project/alien-mollusca-checklist#2]
Originally, the raw data is a compilation of 29 .gdoc files, united in a tabular format as a .gsheet document. Some of the columns in this .gsheets document contain large blocks of text and are thus not very usefull for publication on GBIF. 
The question now is how to maintain documents. There are three options applicable here:
1. Keep the current workflow, i.e. maintain both the .gdoc files and the .gsheet compilation
2. Work directly in the .gsheet compilation and omit the use of the .gdoc files. Keep lengthy text blocks in the .gsheet.
3. Work directly in the .gsheet document and omit all lengthy text blocks. 

Note: there's no option to include the data in a larger database of (alien) molluscs maintained by Thierry as the current database contains many errors and should be updated as well.L

I'm in favour of option 3 as this would greatly simplify our workflow and improven data management. However, this would mean that a lot of contextual information will be lost. So, I guess option 2 will be a better way to include all original information.

**Comment by qgroom** [comment]
Couldn't the full text be put on Zenodo and linked to from the data on GBIF? This would give a citable document with a DOI.
Of course it is down to Thierry, but anything left in a gdoc is going to be temporary.

**Comment by LienReyserhove** [comment]
Good idea, we can put the spreadsheet on Zenodo. I would not publish the .gdoc files as I agreed with Thierry not to use these for future updates, but rather to work in the spreadsheet instead. I would keep the gsheet file alive as it is easy to share and to work in. In case of updates, we could then upload it on Zenodo afterwards.
